src/EnvelopeBuilder.php

```diff
@@ -130,7 +130,7 @@ public function addCarbonCopy(string $name, string $email): self
         $carbonCopy = new Model\CarbonCopy();
         $carbonCopy->setEmail($email);
         $carbonCopy->setName($name);
-        $carbonCopy->setRecipientId((string) $this->docReference);
```

Comment:
Remove useless `$this->docReference` variable

Reply:
mmh; it's still used ^^'

Reply:
My bad!